source/gen/FlashOWare.Generators/Text/Methods.cs

```diff
@@ -2,6 +2,6 @@ namespace FlashOWare.Text;
 
 internal static class Methods
 {
-	internal static string GetName { get; } = "GetName";
-	internal static string IsDefined { get; } = "IsDefined";
+	internal const string GetName  = nameof(Enum.GetName);
+	internal const string IsDefined = nameof(Enum.IsDefined);
```

Comment:
I love the `nameof`-ification here.
And you're right, these are _constants_ ... the `System.Enum.GetName` method group won't change its name.